lib/foreman_ansible/register.rb

```diff
@@ -13,6 +13,14 @@
                  :'api/v2/hostgroups' => [:play_roles,
                                           :multiple_play_roles] },
                :resource_type => 'Hostgroup'
+    permission :show_roles_on_host,
```

Comment:
we usually don't introduce new permission for subset of host/hostgroup attributes or relations, I think just checking `view_hosts` and `view_hostgroups` should be enough